src/applications/vaos/components/tests/VARequestLayout.unit.spec.js

```diff
@@ -56,6 +58,18 @@ describe('VAOS Component: VARequestLayout', () => {
           clinicName: 'Clinic 1',
           clinicPhysicalLocation: 'CHEYENNE',
         },
+        requestedPeriods: [
+          {
+            start: moment(startDate)
+              .subtract(3, 'days')
+              .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+          },
+          {
+            start: moment(startDate)
+              .subtract(4, 'days')
+              .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+          },
+        ],
```

Comment:
Adding a requested date that's in the past to ensure cancel request button still displays